.ci/blackduck_source.sh

```
@@ -1,5 +1,9 @@
 #!/bin/bash -Exel
 
+export GIT_CONFIG_COUNT=1
+export GIT_CONFIG_KEY_0=safe.directory
+export GIT_CONFIG_VALUE_0="${WORSPACE}"
```

Shellcheck (0.11.0)

[info] Possible misspelling: WORSPACE may not be assigned. Did you mean WORKSPACE?

(SC2153)